requirements.txt

```diff
@@ -1,4 +1,5 @@
 appdirs==1.4.2
+docopt==0.6.2
 nose==1.3.7
 packaging==16.8
 pyparsing==2.1.10
```

Comment:
Don't forget to update your requirements.txt using `pip freeze` every time you add new modules to your project.